api/tests/ValidatorTest.php

```diff
@@ -59,4 +59,20 @@ public function testFailingUuidValidation()
 //
 //        $this->fail('An expected exception has not been raised.');
 //    }
+
```

Comment:
@xiphiaz Ah, the country validators needed to be updated to work with the new validation implementation. Done and committed. The `testCamelCaseTransform()` is commented out in master, so I didn't touch that for now. I am actually not sure if that one is needed anymore with the new implementation?